scripts/generator/templates/template-prod-bundle-pom.xml

```diff
@@ -32,6 +32,11 @@
             <version>${project.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin.hilla</groupId>
+            <artifactId>hilla-dev-mode</artifactId>
```

Comment:
This shouldn't go into prod pre-compiled bundle, as it brings HotSwapAgent plugin and vaadin-dev-server

Reply:
I agree. And hilla-dev-mode should be excluded from production builds in general. This is not automatic however but build configs can exclude whole vaadin-dev from the production build.